lib/engine/abilities.rb

```diff
@@ -39,6 +39,10 @@ def remove_ability_when(time)
       end
     end
 
+    def own_abilities
```

Comment:
what is this? can't you just call abilities?

Reply:
Because it was not accessible. I could have made it readable, but the same name is also used in Game as an alias for all_abilities, so I thought I might as well make it more clear. But I can make it that way, if you prefer.

Reply:
yea just add an attr_reader to abilities

Reply:
Done